### PR TITLE
Add p_astro

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This adds a recipe for p_astro.

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
@conda-forge/help-python, I think this recipe is ready for review, thanks.

File: recipes/p_astro/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3df0ab549f07f9de1c9e6b153e4cad65aedaca247d53665a197e78cc8ad80d2a
+  patches:
+    # remove unncessary build dependency
+    - no-pytest-runner.patch
```
Comment (beckermr) [inline recipes/p_astro/meta.yaml]: Let’s remove this patch and simply add pytest-runner to the host section.

File: recipes/p_astro/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - ligo.p_astro
+  requires:
+    - pytest
+
+about:
+  home: http://gracedb.ligo.org
+  doc_url: https://lscsoft.docs.ligo.org/p-astro/
+  license: GPLv3
+  license_family: GPL
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (beckermr) [inline recipes/p_astro/meta.yaml]: Let’s make sure to submit the license upstream.

Comment (duncanmmacleod) [reply recipes/p_astro/meta.yaml]: This has been reported upstream, but the repo is not publicly viewable.

File: recipes/p_astro/meta.yaml (hunk 1 above)
Comment (duncanmmacleod) [reply recipes/p_astro/meta.yaml]: Done (squashed)

**Comment by duncanmmacleod** [comment]
@beckermr, I think I've made the requested changes, thanks.

**Comment by duncanmmacleod** [comment]
Thanks @beckermr!

**Pull request merged**